src/Commands/RunRepeater.ts

```diff
@@ -115,6 +115,8 @@ export class RunRepeater implements CommandModule {
         }
       })
       .option('experimental-connection-reuse', {
```

Comment:
```suggestion
      .option('ntlm', {/* ... */})
      .option('experimental-connection-reuse', {
```
then you have to appropriately set `RequestExecutorOptions` on line 167 handling both parameters and add conflicts clause with those two properties